### PR TITLE
Use Plex API to check remote access down

### PR DESCRIPTION
**Pull request by JonnyWong16**
This should be better now.

### CODE REVIEW

**Comment by JonnyWong16** [comment]
I'm merging into dev so we can test.

**Comment by drzoidberg33** [comment]
Thanks, I'm updating my install.

**Comment by drzoidberg33** [comment]
Bit of a derp on the first run wizard:

screenshot 2015-11-23 11 16 25

File: data/interfaces/default/settings.html [hunk 1]
```diff
@@ -405,9 +405,9 @@
                         </div>
                         <div class="checkbox">
                             <label>
-                                <input type="checkbox" id="monitor_remote_access" name="monitor_remote_access" value="1" ${config['monitor_remote_access']}> Monitor Plex Remote Access
+                                <input type="checkbox" class="monitor-settings" id="monitor_remote_access" name="monitor_remote_access" value="1" ${config['monitor_remote_access']}> Monitor Plex Remote Access
```
Comment (drzoidberg33) [inline data/interfaces/default/settings.html:408]: This doesn't require a restart.

Just need to restart the scheduler:

`plexpy.initialize_scheduler()`

File: data/interfaces/default/settings.html [hunk 2]
```diff
@@ -479,7 +479,12 @@
                         </div>
                         <div class="checkbox">
                             <label>
-                                <input type="checkbox" name="movie_notify_enable" id="movie_notify_enable" value="1" ${config['movie_notify_enable']}> Enable Movie and TV Notifications
+                                <input type="checkbox" name="movie_notify_enable" id="movie_notify_enable" value="1" ${config['movie_notify_enable']}> Enable Movie Notifications
+                            </label>
+                        </div>
+                        <div class="checkbox">
+                            <label>
+                                <input type="checkbox" name="tv_notify_enable" id="tv_notify_enable" value="1" ${config['tv_notify_enable']}> Enable TV Show Notifications
```
Comment (drzoidberg33) [inline data/interfaces/default/settings.html:487]: You need to enable this after an update if the user had the old global video notifications option enabled. After updating on my system it disabled TV notifications, people going to wonder why their notifications are broken.

File: data/interfaces/default/settings.html (hunk 1 above)
Comment (JonnyWong16) [inline data/interfaces/default/settings.html:408]: Actually it doesn't require either?

The scheduled task already runs, and the setting just toggles that if statement.

File: data/interfaces/default/settings.html (hunk 2 above)
Comment (JonnyWong16) [inline data/interfaces/default/settings.html:487]: ~~Not sure where or how I would do this.~~

The original Headphones code uses `CONFIG_VERSION` to keep track of upgrades. I'll try that.

**Pull request merged**